### ISSUE TITLE
Create PHPStorm Codestyle setting file for common CodeStyles

### ISSUE THREAD

**Issue by kremsy** [joeldg/bowhead#39]
Would be great if there could be a common Codestyle that all looks the same, and phpstorm autostyle will not mess whole files which are commited later.

**Comment by rxmg-joeldg** [comment]
good idea, my phpstorm file is kind of nuts for a lot of people because I used vim keybindings

**Comment by kremsy** [comment]
I have a quite good one (https://github.com/ManiaControl/ManiaControl/blob/master/.idea/codeStyleSettings.xml) , but I guess I need to modify it according to get it with most of the projects code in accordance, for example I generally use opening Brackets in same line:

`    public function bowhead_rsi_macd($pair, $data, $return_full=false){`

instead of 

```
    public function bowhead_rsi_macd($pair, $data, $return_full=false)
{
```

Will take a look into changing these minor things, and than I can provide the codestylesetting file.
